static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -276,34 +265,29 @@ var Crossword = React.createClass({
     },
 
     check: function (entry) {
-        var that = this,
-            cells = _.map(helpers.cellsForEntry(entry), function (cell) {
-                return that.state.grid[cell.x][cell.y];
-            }),
+        var cells = _.map(helpers.cellsForEntry(entry), (cell) => this.state.grid[cell.x][cell.y]),
             badCells;
 
         if (entry.solution) {
-            badCells = _.map(_.filter(_.zip(cells, entry.solution), function (cellAndSolution) {
+            badCells = _.map(_.filter(_.zip(cells, entry.solution), (cellAndSolution) => {
                 var cell = cellAndSolution[0],
                     solution = cellAndSolution[1];
                 return /^[A-Z]$/.test(cell.value) && cell.value !== solution;
```

Comment:
I think you do still need return here? My understanding was you can only do without return for one line/expression functions.